src/registry.js

```diff
@@ -0,0 +1,296 @@
+"use strict";
```

Comment:
The changes to the original polyfill were things like this:

```js
const origHTMLElement = win.HTMLElement;
// TO
const origHTMLElement = domino.impl.HTMLElement;
```

which were contained [in the same file](https://github.com/webcomponents/custom-elements/blob/master/src/custom-elements.js). Are you suggesting we move this to a separate file? Sorry if I'm still misunderstanding.